index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Hello Microverse</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <h1>Hello Microverse!</h1>
+  </body>
+</html>
```

Comment:
Hi @levy002 I have tried to go through the requested changes and everything looks fine only the gitignore, test.md and readme.md files were accidentally sent to the main branch. Perhaps you can check them out and advise accordingly and guide me on the way forward. I seem stuck.